tests/Unit/Momento.Sdk.Tests/CacheClientConstructorTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Momento.Sdk;
+using Momento.Sdk.Auth;
+using Momento.Sdk.Config;
+using Xunit;
+
+public class CacheClientConstructorTest
+{
+    const String TEST_AUTH_TOKEN = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJmb29Abm90LmEuZG9tYWluIiwiY3AiOiJjb250cm9sLXBsYW5lLWVuZHBvaW50Lm5vdC5hLmRvbWFpbiIsImMiOiJjYWNoZS1lbmRwb2ludC5ub3QuYS5kb21haW4ifQo.rtxfu4miBHQ1uptWJ2x3UiAwwJYcMeYIkkpXxUno_wIavg4h6YJStcbxk32NDBbmJkJS7mUw6MsvJNWaxfdPOw";
```

Comment:
Shouldn't this be an environment variable or some other substituted value so we don't check an auth token into source control?